yearn/deploy/00_deploy_cvp_strategy.ts

```diff
@@ -0,0 +1,49 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts, helpers } = hre
+  const { deploy, log } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  const TBTCCurveVault = await deployments.getOrNull("TBTCCurveVault")
+  if (!TBTCCurveVault || !helpers.address.isValid(TBTCCurveVault.address)) {
```

Comment:
What if we use `get` instead of `getOrNull`? `get` should throw an error if the deployment is not found.

```suggestion
  const TBTCCurveVault = await deployments.get("TBTCCurveVault")
  if (!helpers.address.isValid(TBTCCurveVault.address)) {
```

Reply:
Good point! Please see: https://github.com/keep-network/tbtc-v2/pull/14/commits/96b6813293702d9bec6aaa5f0af1f75d6c057ad8